models/tridimensional/docking_validation/config.py

```diff
@@ -0,0 +1,89 @@
+#      /--- PDB ---/   pdb files (protein-dna structure) go here         #
+# BASE /- RESULTS -/   results and simulation scores go here             #
+#      /- SCRIPTS -/   all scripts go here (should be flat)              #
+# ====================================================================== #
+
+DIR_PDB = os.path.join(DIR_BASE, "pdb")
+DIR_RESULTS = os.path.join(DIR_BASE, "results")
+DIR_SCRIPTS = os.path.join(DIR_BASE, "scripts")
+
+
+# ====================================================================== #
+# PDB:                                                                   #
+# ====================================================================== #
+#                                                                        #
+#     /- special -/    any special pdb files, esp. the original pdb      #
+#     |                                                                  #
+# pdb /- mutants -/-- mutant_0 --/-- template --/   1 pdb                #
+#                 |              /-- variants --/   256 pdbs             #
+#                 |                                                      #    
+#                 /-- mutant_1 --/-- template --/   1 pdb                #
+#                 |              /-- variants --/   256 pdbs             #
+#                 |                                                      #
+#                 etc.                                                   #
+#                                                                        #
+# Every mutant gets its owned subdirectory of pdb files based on index.  #
+# For example, if index=12, then the mutant has a template associated    #
+# with applying the index=12 mutations from our mutant database to       #
+# the original pdb file in  "pdb/special/".                              #
+# This template pdb is left in:  "pdb/mutants/mutant_12/template/"       #
+# Once the template is created, 64 or 256 pam variants are created and   #
+# stored in:  "pdb/mutants/mutant_12/variants/".                         #
+# ====================================================================== #
+
+DIR_PDB_MUTANTS = os.path.join(DIR_BASE, "mutants")
+DIR_PDB_SPECIAL = os.path.join(DIR_BASE, "special")
+PATH_PDB_SPECIAL_ORIGINAL = os.path.join(DIR_PDB_SPECIAL, "4UN3.original.pdb")
+PATH_PDB_SPECIAL_VQR = os.path.join(DIR_PDB_SPECIAL, "4UN3.mutant_VQR.pdb")
+PATH_PDB_SPECIAL_EQR = os.path.join(DIR_PDB_SPECIAL, "4UN3.mutant_EQR.pdb")
+FOLDERNAME_MUTANT_TEMPLATE = "template"
+FOLDERNAME_MUTANT_VARIANTS = "variants"
+MUTANT_TEMPLATE_PREFIX = "4UN3.mutant_"
+
+
+# ====================================================================== #
+# RESULTS:                                                               #
+# ====================================================================== #
+#                                                                        #
+#         /- batch -/   lone validation runs in labelled folders         #
```

Comment:
validation runs for a particular purpose/to check a particular things ("lone" confusing)